docker/.env.default

```diff
@@ -14,6 +14,9 @@ JWT_SECRET=bf549f5188556ce0951048ef467ec93067bc4ea21acebe46ef675cd4e8e015ff
 # Magi's external rpc service port
 RPC_PORT=9545
 
+# Magi's external rpc service socket address
+RPC_ADDR="0.0.0.0"
```

Comment:
note - this is intentionally set to 0.0.0.0 for the docker .env file, so that the port publishing in the docker compose file works as expected and the RPC can be accessed by the host.